hackathon_site/dashboard/frontend/src/components/general/ProductOverview/ProductOverview.js

```diff
@@ -0,0 +1,196 @@
+import React from "react";
+import Typography from "@material-ui/core/Typography";
+import Link from "@material-ui/core/Link";
+import Button from "@material-ui/core/Button";
+import MenuItem from "@material-ui/core/MenuItem";
+import FormControl from "@material-ui/core/FormControl";
+import Select from "@material-ui/core/Select";
+import LaunchIcon from "@material-ui/icons/Launch";
+import InputLabel from "@material-ui/core/InputLabel";
+import Chip from "@material-ui/core/Chip";
+import styles from "./ProductOverview.module.scss";
+import Alert from "@material-ui/lab/Alert";
+
+import { Formik } from "formik";
+
+const createQuantityList = (number) => {
+    let entry = [];
+
+    for (let i = 1; i <= number; i++) {
+        let valueInString = i + "";
+        entry.push(
+            <MenuItem key={i} role="quantity" value={valueInString}>
+                {i}
+            </MenuItem>
+        );
+    }
+
+    return entry;
+};
+
+export const CartForm = ({
+    availableQuantity,
+    handleSubmit,
+    handleChange,
+    requestFailure,
+    values: { quantity },
+}) => {
+    if (!handleSubmit) {
```

Comment:
This makes it hard to work on the component while developing, it can be removed. We'll know pretty quickly when an error gets thrown that `handleSubmit` is undefined.

Reply:
I had this because nothing renders if I don't return false